### ISSUE TITLE
IDFix tool _Offline installation

### ISSUE THREAD

**Issue by YanivLalush** [microsoft/idfix#32]
HI,

Is there any option to run the tool in offline mode with out connecting first to the internet?
Thanks.

**Comment by fred-pe** [comment]
Hi,

The setup is only used to check the prerequisites.
It is possible to pass parameters to it (do "setup -h" to display them).
I downloaded the files to my computer, then I copied them to a server that did not have internet access. By setting the various parameters to point them to the directory in which I had deposited the files, the prerequisite test is passed.
On the other hand afterwards it tries to run "IdFix.application" to make the effective installation and that I did not manage to make it work.
But by directly executing "IdFix.exe", it works.

Regards,
Fred
